libs/platform/src/lib/components/form/input/input.component.html

```diff
@@ -12,7 +12,13 @@
     [attr.placeholder]="placeholder"
     [disabled]="disabled"
     [compact]="contentDensity == 'compact'"
-    [state]="status === 'error' ? 'invalid' : ''"
+    [state]="status"
+    [readOnly]="readonly"
     (blur)="_onFocusChanged(false)"
     (focus)="_onFocusChanged(true)"
+    [attr.aria-disabled]="disabled"
```

Comment:
Do we need `aria-disabled` if we already are using the `disabled` attribute?

Reply:
yes, this allows us to readout input field whether to identify disabled or readonly over voice.